Exception in thread "Compose Recompiler" java.io.IOException: Cannot run program "gradlew.bat"

When trying to run `desktopRun` on Windows, the following exception is logged

```
Exception in thread "Compose Recompiler" java.io.IOException: Cannot run program "gradlew.bat" (in directory "C:\Users\Kirill\IdeaProjects\MyProject"): CreateProcess error=2, Das System kann die angegebene Datei nicht finden
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1170)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1089)
	at org.jetbrains.compose.reload.agent.RecompilerKt.launchRecompiler$lambda$6(recompiler.kt:50)
	at kotlin.concurrent.ThreadsKt$thread$thread$1.run(Thread.kt:30)
Caused by: java.io.IOException: CreateProcess error=2, Das System kann die angegebene Datei nicht finden
	at java.base/java.lang.ProcessImpl.create(Native Method)
	at java.base/java.lang.ProcessImpl.<init>(ProcessImpl.java:500)
	at java.base/java.lang.ProcessImpl.start(ProcessImpl.java:159)
	at java.base/java.lang.ProcessBuilder.start(ProcessBuilder.java:1126)
	... 3 more

```

--- a/hot-reload-agent/src/main/kotlin/org/jetbrains/compose/reload/agent/recompiler.kt
+++ b/hot-reload-agent/src/main/kotlin/org/jetbrains/compose/reload/agent/recompiler.kt
@@ -35,9 +35,15 @@ internal fun launchRecompiler() {
     val recompilerThread = thread(name = "Compose Recompiler") {
         logger.debug("'Compose Recompiler' started")
 
+        val gradleScriptCommand = if ("win" in System.getProperty("os.name").lowercase()) {
+            arrayOf("cmd", "/c", "start", "gradlew.bat")
+        } else {
+            arrayOf("./gradlew")
+        }
+
         val process = ProcessBuilder().directory(File(composeBuildRoot))
             .command(
-                if ("win" in System.getProperty("os.name").lowercase()) "gradlew.bat" else "./gradlew",
+                *gradleScriptCommand,
                 createGradleCommand(composeBuildProject, composeBuildCompileTask),
                 "--console=plain",
                 "--no-daemon",

